model/qti/parser/IncludedElementIdsExtractor.php

```diff
@@ -61,7 +62,7 @@ public function withMediaResolver(TaoMediaResolver $mediaResolver): self
     private function getMediaResolver(): TaoMediaResolver
     {
         if (!$this->mediaResolver) {
-            $this->mediaResolver = new TaoMediaResolver();
+            $this->mediaResolver = new ItemMediaResolver(null, '');
```

Comment:
I kept the `TaoMediaResolver()` and I still can import item using shared stimulus. How could you simulate the error?

Reply:
To be in the safe side, can you also add a try/catch "inside" the foreach of the "extract" method. I could not simulate the error, so I do not trust this method anymore: https://github.com/oat-sa/extension-tao-itemqti/pull/1461/files#diff-904859110fe51318fa9e992399bdcf25L44

- [  ] Add try / catch

Reply:
we shouldnt because we need to stop the import and rollback if smth happens. If the import is successful but assets are missing, it is going to create false positive